### ISSUE TITLE
Process MUTO-INTL Alignment Workflow

### ISSUE THREAD

**Issue by b-f-chan** [icgc-argo/workflow-roadmap#293]
Alignment worksheet: https://docs.google.com/spreadsheets/d/13uxLJEjv5m6Q4nNOAsgeIKs8gm2rMctBK2CDXL5c4lU/edit#gid=834210998

Status - 1010 samples in total

**Comment by b-f-chan** [comment]
@lindaxiang may require @UmmulkiramR help to process this.  Adding to next ARGO sprint as a candidate (cc: @sifavahora @Buwujiu)

**Comment by b-f-chan** [comment]
@lindaxiang may require @UmmulkiramR help to process this.  Adding to next ARGO sprint as a candidate (cc: @sifavahora @Buwujiu)

**Comment by lindaxiang** [comment]
The plan is to use Model Tee to submit the workflow jobs to workflow scheduler.

**Comment by Buwujiu** [comment]
ON hold for now, muto has to resubmit data.

**Comment by Buwujiu** [comment]
@hknahal will follow up if MUTO's submission status.

**Comment by Buwujiu** [comment]
blocked by prod issue https://github.com/icgc-argo/workflow-roadmap/issues/319. @lindaxiang please send me the link

**Comment by Buwujiu** [comment]
MUTO-INTL molecular submission is blocked by score upload issue. Need to fix score first.

**Comment by Buwujiu** [comment]
@lindaxiang need to talk to Mutl-INTL and ask them to submit data.

**Comment by lindaxiang** [comment]
~10% out of total 1010 alignments completed in ~1 week.

**Comment by lindaxiang** [comment]
346/1010 aln completed

**Comment by edsu7** [comment]
730/1010 completed.

**Comment by lindaxiang** [comment]
952/1010 completed

**Comment by lindaxiang** [comment]
All 1010 samples alignment have completed. This ticket can be closed. 
Next step: open new ticket to QC the results.

**Comment by lindaxiang** [comment]
QC ticket has been created: https://github.com/icgc-argo/workflow-roadmap/issues/398
